jest.setup.ts

```diff
@@ -35,6 +35,14 @@ jest.mock("./src/types/settings.d", () => ({
   },
 }));
 
+jest.mock("./src/types/settings.d", () => ({
+  SortOrder: {
+    ASC: "A-Z",
+    DESC: "Z-A",
+    NONE: "none",
+  },
+}));
+
```

Comment:
above is a mock for the same file, put this new object on that mock instead of mocking it again:

![image](https://github.com/user-attachments/assets/e46766d6-4d39-44ef-8521-22a193968f02)